<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="it-it">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-317e149" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">bd7cba635c93de4751058aba7b65c1eff2217e2e</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">docs\visual-basic\misc\bc42031.md</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">vbcs_lo-mt</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Machine Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02da59eeea18790d8d43ea1a1f842c5fb5b37e94</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">88f59cd13f226a3d687b2f970278d873c7934413</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>&amp;#39;Catch&amp;#39; block never reached; <ph id="ph1">&amp;lt;</ph>exception<ph id="ph2">&amp;gt;</ph> handled above in the same &amp;#39;Try&amp;#39; statement | Microsoft Docs</source>
          <target state="translated">Blocco 'Catch' mai raggiunto. <ph id="ph1">&amp;lt;</ph>eccezione<ph id="ph2">&amp;gt;</ph> gestito in precedenza nella stessa istruzione "Try" | Documenti di Microsoft</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve">
          <source>&amp;#39;Catch&amp;#39; block never reached; <ph id="ph1">&amp;lt;</ph>exception<ph id="ph2">&amp;gt;</ph> handled above in the same &amp;#39;Try&amp;#39; statement</source>
          <target state="translated">Blocco 'Catch' mai raggiunto. <ph id="ph1">&amp;lt;</ph>eccezione<ph id="ph2">&amp;gt;</ph> gestito in precedenza nella stessa istruzione "Try"</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve">
          <source>A <ph id="ph1">`Catch`</ph> block in the code cannot be reached because it is handled in a preceding <ph id="ph2">`Try`</ph> block.</source>
          <target state="translated">Un blocco <ph id="ph1">`Catch`</ph> nel codice non è raggiungibile perché è gestito in un blocco <ph id="ph2">`Try`</ph> precedente.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve">
          <source>By default, this message is a warning.</source>
          <target state="translated">Per impostazione predefinita, si tratta di un messaggio di avviso.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve">
          <source>For more information about hiding warnings or treating warnings as errors, please see <bpt id="p1">[</bpt>Configuring Warnings in Visual Basic<ept id="p1">](https://docs.microsoft.com/visualstudio/ide/configuring-warnings-in-visual-basic)</ept></source>
          <target state="translated">Per ulteriori informazioni su come nascondere gli avvisi o considerarli come errori, vedere <bpt id="p1">[</bpt>configurazione degli avvisi in Visual Basic<ept id="p1">](https://docs.microsoft.com/visualstudio/ide/configuring-warnings-in-visual-basic)</ept></target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve">
          <source><bpt id="p1">**</bpt>Error ID:<ept id="p1">**</ept> BC42031</source>
          <target state="translated"><bpt id="p1">**</bpt>ID errore:<ept id="p1">**</ept> BC42031</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve">
          <source>To correct this error</source>
          <target state="translated">Per correggere l'errore</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve">
          <source>Remove the redundant statement.</source>
          <target state="translated">Rimuovere l'istruzione ridondante.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve">
          <source>See Also</source>
          <target state="translated">Vedere anche</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve">
          <source><bpt id="p1">[</bpt>How to: Catch an Exception in Visual Basic<ept id="p1">](http://msdn.microsoft.com/en-us/f3063c89-d2bf-49b1-91b5-b87edfb18b95)</ept><ph id="ph1"> </ph></source>
          <target state="translated"><bpt id="p1">[</bpt>Procedura: intercettare un'eccezione in Visual Basic<ept id="p1">](http://msdn.microsoft.com/en-us/f3063c89-d2bf-49b1-91b5-b87edfb18b95)</ept><ph id="ph1"> </ph></target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve">
          <source><bpt id="p1"> [</bpt>How to: Test Code with a Try…Catch Block in Visual Basic<ept id="p1">](http://msdn.microsoft.com/en-us/8368e205-ed73-4185-a247-af84fb4fafa9)</ept><ph id="ph1"> </ph></source>
          <target state="translated"><bpt id="p1"> [</bpt>Procedura: testare il codice con un blocco Try... Blocco catch in Visual Basic<ept id="p1">](http://msdn.microsoft.com/en-us/8368e205-ed73-4185-a247-af84fb4fafa9)</ept><ph id="ph1"> </ph></target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve">
          <source><bpt id="p1"> [</bpt>How to: Filter Errors in a Catch Block in Visual Basic<ept id="p1">](http://msdn.microsoft.com/en-us/85964d0a-56e7-4301-a96e-5eaea23b7b9b)</ept><ph id="ph1"> </ph></source>
          <target state="translated"><bpt id="p1"> [</bpt>Procedura: filtrare gli errori in un blocco Catch in Visual Basic<ept id="p1">](http://msdn.microsoft.com/en-us/85964d0a-56e7-4301-a96e-5eaea23b7b9b)</ept><ph id="ph1"> </ph></target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve">
          <source><bpt id="p1"> [</bpt>Walkthrough: Structured Exception Handling (Visual Basic)<ept id="p1">](http://msdn.microsoft.com/en-us/440da655-4b32-490b-8b16-bfe46f41fa76)</ept><ph id="ph1"> </ph></source>
          <target state="translated"><bpt id="p1"> [</bpt>Procedura dettagliata: Gestione strutturata delle eccezioni (Visual Basic)<ept id="p1">](http://msdn.microsoft.com/en-us/440da655-4b32-490b-8b16-bfe46f41fa76)</ept><ph id="ph1"> </ph></target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve">
          <source><bpt id="p1"> [</bpt>Try...Catch...Finally Statement<ept id="p1">](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)</ept></source>
          <target state="translated"><bpt id="p1"> [</bpt>Istruzione Try...Catch...Finally<ept id="p1">](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)</ept></target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>